homework-submission/todo-app/src/todos/element.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faCheck, faTrash, faPencilAlt, faTimes } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { fab } from '@fortawesome/free-brands-svg-icons'
+
+ 
+library.add(fab, faCheck, faTrash, faPencilAlt, faTimes);
+
+class Element extends Component {
+    state = {
+        id: this.props.id,
+        description: this.props.description,
+        deadline: this.props.deadline,
+        done: this.props.done,
+        editMode: false,
+        temporarySave: this.props.description
+     }
+
```

Comment:
you don't need a new state here because it will not update the basic state when you update it

Reply:
well , i update the description state when i add a new todo